SPECS-EXTENDED/selinux-policy-ci/selinux-policy-ci.spec

```diff
@@ -0,0 +1,104 @@
+%global modulename hotfix
+
+Summary:        Azure Linux CI SELinux policy
+Name:           selinux-policy-ci
```

Comment:
I see Steamboat also carries selinux-policy package. Do we need that here as well?